photo_mapper_webserver/authentication/security.py

```
@@ -0,0 +1,156 @@
+class CookieJWTAuth:
+    def __call__(self, request: HttpRequest):
+        token = request.COOKIES.get(
+            settings.ACCESS_COOKIE_NAME
+        )
+
+        if not token:
+            return None
+
+        try:
+            return get_user_from_access_token(token)
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)